api.md

```diff
@@ -170,6 +172,66 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## attach_data_node() [](attach_data_node)
+
+Attach a data node to a hypertable. The data node should have been
+previously created using [`add_data_node`](#add_data_node).
+
+When a distributed hypertable is created it will by default use all
+available data nodes for the hypertable, but if a data node is added
+*after* a hypertable is created, the data node will not automatically
+be used by existing distributed hypertables.
+
+If you want a hypertable to use a data node that was created later,
+you need to attach the data node to the hypertable using this
+function. This function *MUST* be run on an access node.
+
+#### Required Arguments [](attach_data_node-required-arguments)
+
+| Name              | Description                                   |
+| `hypertable`      | Name of hypertable to attach node to          |
+| `node_name`       | Local name of data node to attach             |
+
+##### Parameter Description
+
+* `hypertable` - The name of the hypertable where the data node should
+  be attached.
+
+* `node_name` - Local name of the data node to attach to the
+  hypertable.
+
+#### Optional Arguments [](attach_data_node-optional-arguments)
+
+| Name              | Description                                   |
+| `if_not_attached` | Do not error if data node is already attached |
+
+##### Parameter Description
+
+* `if_not_attached` - Do not generate an error if the data node is
+  already attached to the hypertable. A notice will be printed that
+  the data node is attached.
+
+#### Returns
+
+| Column               | Description                              |
+| `hypertable_id`      | Hypertable id of the modified hypertable |
+| `node_hypertable_id` | Hypertable id on the remote data node    |
+| `node_name`          | Local name of the attached data node     |
+
+#### Sample Usage [](attach_data_node-examples)
+
+Let us assume that you have a distributed hypertable `conditions`
```

Comment:
Is this too verbose?

"Attach a data node `dn3` to distributed hypertable `conditions`" and take out the `add_data_node` command in the code block.  Also, is there any output?

Reply:
Okay, fix here was to say that you should have added and attached the data node already

Reply:
Actually, there is some complexity here, and I think it's nice to clarify that the node should be added before it's attached.

Reply:
Output added